Tools/WinMLDashboard/README.md

```diff
@@ -75,13 +75,16 @@ To debug your model follow these steps:
 
 <img src="./public/ConfigureDebug.png" width=800/>
 <img src="./public/DebugView.png" width=800/>
+<img src="./public/DebugPngs.png" width=800/>
```

Comment:
Can you add like one-line description before showing the debug output?

Reply:
There is a one line description under the image. Would you prefer that to be above the image?

Reply:
I think so. Separate the output image and the image of how to use above with words might be a better idea.